spacesuit/src/value.rs

```diff
@@ -120,6 +120,7 @@ impl SignedInteger {
         }
     }
 
+    /// Returns the Scalar representation of self.
```

Comment:
Pedant mode: Scalar is not strictly speaking a representation of integer. You can’t meaningfully convert back into u64, unless you knew that it never wrapped around after converting from u64.

How about?:

```suggestion
    /// Converts the integer to Scalar.
```